tools/autograd/derivatives.yaml

```diff
@@ -963,6 +964,7 @@
 - name: kl_div_backward(Tensor grad_output, Tensor self, Tensor target, bool size_average, bool reduce)
   grad_output: kl_div_double_backward_grad_output(grad, self, target, size_average, reduce)
   self: zeros_like(grad)
+  target: zeros_like(grad)
```

Comment:
What does this do?

Reply:
oops, will remove it. Yes, it was for double-backward. But sounds like I don't need that.

Reply:
If it computes the correct ggTarget, why should we remove it?

Reply:
ah, I see, it makes sense, I will keep it then.